recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,11 @@
+# c_compiler:        # [win]
+#   - vs2015   # [win and py3k]
+#   - vs2015   # [win and py3k]
+#   - vs2008   # [win and py2k]
+# cxx_compiler:        # [win]
+#   - vs2015   # [win and py3k]
+#   - vs2015   # [win and py3k]
+#   - vs2008   # [win and py2k]
+zstd:        # [win and py2k]
```

Comment:
I don't think `py27` selector works. Why don't you hard code the version in `meta.yaml` and remove this file?

Reply:
good idea!